src/eggd_CANRISK_vcf.sh

```
@@ -84,8 +84,7 @@ main() {
 
     mark-section "Checking for low coverage"
     # identify variants with low coverage (below depth threshold input)
-    filter="FORMAT/DP<$depth"
-    bcftools filter -i "$filter" "$sample_vcf_path" | \
+    bcftools filter -i "FORMAT/DP<$depth && CHROM!='X'" "$sample_vcf_path" | \
```

Shellcheck (0.10.0)

[warning] depth is referenced but not assigned.

(SC2154)